trie/verkle.go

```diff
@@ -465,9 +455,14 @@ func (t *VerkleTrie) UpdateContractCode(addr common.Address, codeHash common.Has
 
 		// Reuse the calculated key to also update the code size.
 		if i == 0 {
-			cs := make([]byte, 32)
-			binary.LittleEndian.PutUint64(cs, uint64(len(code)))
-			values[utils.CodeSizeLeafKey] = cs
+			headervals, err := t.root.(*verkle.InternalNode).GetValuesAtStem(key[:31], t.FlatdbNodeResolver)
+			if err != nil {
+				return fmt.Errorf("UpdateContractCode (addr=%x) error while getting account header: %w", addr[:], err)
+			}
+			var cs [8]byte
+			binary.LittleEndian.PutUint64(cs[:], uint64(len(code)))
+			copy(headervals[utils.BasicDataLeafKey][utils.BasicDataCodeSizeOffset:], cs[5:8])
```

Comment:
Is this `c[5:8]` wrong? Shouldn't be `c[:3]` since `cs` was written with `binary.LittleEndian`.

Also, I'd suggest as said before avoiding magic numbers, i.e: `c[:utils.CodeSizeSize]` or similar.

Reply:
yup, good catch, it's the target that should be `[5:8]`